eisen_extras/torchvision/transforms.py

```diff
@@ -0,0 +1,214 @@
+import numpy as np
+import torch
+from typing import List
+
+from PIL.Image import Image
+from torchvision import transforms as tvt
+
+
+class PILImageTorchvisionTransforms:
+    """# TODO Torchvision transforms for PIL Image input.
+    """
+    def __init__(self, fields: List[str], transform):
+        self.fields = fields
+        self.transform = transform
+
+    def __call__(self, data: dict) -> dict:
+        for field in self.fields:
+            self._check_data_type(data[field])
+            data[field] = self.transform(data[field])
+        return data
+
+    def _check_data_type(self, instance):
+        """# TODO"""
+        if not isinstance(instance, Image):
+            raise TypeError('Input must be of type PIL Image. Type %s is the current input.' % type(instance))
+
+
+class CenterCrop(PILImageTorchvisionTransforms):
+    """# TODO"""
+    def __init__(self, fields: List[str], *args, **kwargs):
```

Comment:
This is good. it probably already works fine (this holds for all of the transforms here). We will need some tests and docstrings.

Reply:
Yes, the docstrings and test cases are still missing here. Will implement those in the next.